csharp/ql/test/library-tests/csharp8/PrintAst.expected

```diff
@@ -1047,45 +1047,49 @@ ranges.cs:
 #   20|           1: [LocalVariableAccess] access to local variable slice8
 #   25| [NamespaceDeclaration] namespace ... { ... }
 #   27|   1: [Struct] Index
-#   29|     5: [InstanceConstructor] Index
+#-----|     3: (Base types)
+#   27|       1: [Interface] IEquatable<Index>
```

Comment:
I was not expecting this change. Were interfaces of value types not extracted before? I'm not sure we would want to see these. I'll check later if they can be marked as compiler generated or not.

Reply:
Good catch! Turns out that this is because `Index` [is defined in source code as well](https://github.com/github/codeql/blob/main/csharp/ql/test/library-tests/csharp8/ranges.cs#L24-L42), and now the two entities are mapped to one. I will remove the source code definition, as it is no longer needed.